spyderlib/widgets/sourcecode/codeeditor.py

```diff
@@ -915,6 +915,28 @@ def fix_indentation(self):
             self.setPlainText(text_after)
             self.document().setModified(True)
 
+    def open_rename_variable_dialog(self):
+        """
+        Open rename variable dialog box and return result.
+
+        Returns (source_code, offset, new_name) on success, None otherwise.
+        """
+        from spyderlib.qt.QtGui import QMessageBox, QInputDialog
+        old_name = self.get_current_word()
+        msg = "Enter new name for '{}':".format(old_name)
+        new_name, ok = QInputDialog.getText(self, 'Rename', msg, text=old_name)
+        if ok and new_name and new_name != old_name:
+            source_code = to_text_string(self.toPlainText())
+            offset = self.get_position('cursor')
+            return (source_code, offset, new_name)
+        else:
+            return None

```

Comment:
`else: return None` feels strange. Removing these lines would work, but just removing `else` and deindenting the return would be clearer.